JYP-iOS/JYP-iOS/Sources/CompositionRoot.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  CompositionRoot.swift
+//  JYP-iOS
+//
+//  Created by 송영모 on 2022/12/28.
+//  Copyright © 2022 JYP-iOS. All rights reserved.
+//
+
+import UIKit
+
+struct AppDependency {
+    let window: UIWindow
+    let configureSDKs: () -> Void
+    let configureAppearance: () -> Void
+}
+
+class CompositionRoot {
+    static func resolve(windowScene: UIWindowScene) -> AppDependency {
+        let window = UIWindow(windowScene: windowScene)
+        window.backgroundColor = .white
+        window.makeKeyAndVisible()
+        
+        window.rootViewController = makeTabBarScreen()
```

Comment:
```suggestion
        window.rootViewController = UINavigationController(rootViewController: makeTabBarScreen())
```
추가로, window의 root에 navigationController를 까는 게 어떨까 제안드립니다. 이렇게 변경했을 경우 44~48 line에서 일일히 navigationController로 감싸줄 필요가 없어질 것 같습니다.

Reply:
질문) 이렇게 바꿨을때 2번 탭에서 뎁스가 깊어진 뷰컨에서 popToRootViewController를 하면 다시 2번 탭 맨 처음으로 돌아갈 수 있나요 ? (1번 탭, 2번 탭, 3번 탭이 rootVC가 될수있는지가 궁금합니다..!)

Reply:
아 이 부분은 제가 잘못 생각한 것 같습니다. 영모님이 지적하신 부분이 맞습니다!!